cips/cip-29.md

```diff
@@ -49,30 +49,30 @@ Implementers MUST ensure:
-| **9**   | 3.10               | 2.77                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **10**  | 2.79               | 2.58                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **11**  | 2.51               | 2.41                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **12**  | 2.26               | 2.25                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **13**  | 2.03               | 2.10                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **14**  | 1.83               | 1.96                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **15**  | 1.65               | 1.83                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **16**  | 1.50               | 1.70                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **17**  | 1.50               | 1.59                   | Regular annual disinflation applied (6.7%).                                                                                  |
-| **18**  | 1.50               | 1.50                   | Regular annual disinflation applied (6.7%).                                                                                  |
+Below is the illustrative table showing the original versus the accelerated schedule. The "New Adjusted Inflation" column is an example of how inflation might look if we reduce it by 33% starting in the next upgrade. Note that for simplicity, the table assumes the inflation drop is applied at year 1.5 but if this proposal gets accepted, it would be immediately applied with the next upgrade.
+
+|    Year | Original Inflation | New Adjusted Inflation | Notes                                                                   |
+|--------:|-------------------:|-----------------------:|:------------------------------------------------------------------------|
+|   **0** |               8.00 |                   8.00 | Genesis year, no change.                                                |
+|   **1** |               7.20 |                   7.20 |                                                                         |
+| **1.5** |               7.20 |                5.00088 | Reduce year 0 inflation by 33%, lowering the inflation rate to 5.00088% |
```

Comment:
Shouldn't this be "Reduce year *1* inflation by 33%" still?

Reply:
The revised proposal is to preserve this formula at all times:

```
InitialInflationRate * ((1 - DisinflationRate) ^ YearsSinceGenesis)
```
for versions 1 through 3:
```
InitialInflationRate = 0.08
DisinflationRate = 0.1
```
for versions 4 and up:
```
InitialInflationRateCip29 = 0.0536
DisinflationRateCip29 = 0.067
```

so you're right on how to calculate the year 1.5 inflation. For years after that, you can use the new constants with the original formula.